packages/core/src/amazonqGumby/mock/downloadHilZip/manifest.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
should this be in `packages/core/src/testFixtures/` , or somewhere else in  `src/test/`